nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/marshallers/json/document/DocumentPropertiesJsonReader.java

```diff
@@ -200,14 +206,17 @@ private Object getScalarPropertyValue(Property property, JsonNode jn, Type type)
                         return blob.getString();
                     }
                 }
-                log.info("Unable to parse the property {}", property::getXPath);
-                return null;
+                throw new MarshallingException("Unable to parse the property " +  property.getXPath());
```

Comment:
I assume this is caught by the new `catch (Exception` further down? Can't we be more specific in the catch?

Reply:
Exactly, this is caught by the former `catch (Exception e)`. It has been replaced by `catch (MarshallingException e)`.